azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/controlbar/ControlBarViewModel.kt

```diff
@@ -216,4 +269,16 @@ internal class ControlBarViewModel(
     private fun dispatchAction(action: Action) {
         dispatch(action)
     }
+
+    private fun callOnClickHandler(
+        context: Context,
+        buttonOptions: CallCompositeButtonOptions?,
+    ) {
+        try {
+            buttonOptions?.onClickHandler?.handle(
+                createButtonClickEvent(context, buttonOptions)
+            )
+        } catch (_: Exception) {
```

Comment:
Should Log.e so developer can see error we are squelching in Logcat.

Reply:
Added